ranger-core/src/main/java/io/appform/ranger/core/finder/serviceregistry/ServiceRegistryUpdater.java

```diff
@@ -70,6 +73,8 @@ public void start() {
         try {
             RetryerBuilder.<Boolean>newBuilder()
                     .retryIfResult(r -> null == r || !r)
+                    .retryIfException()
+                    .withWaitStrategy(WaitStrategies.fixedWait(1, TimeUnit.SECONDS))
```

Comment:
Don't think the sleepTime should be hardcoded. Can we move this to an arg, in BaseServiceFinderBuilder where the updater is created, much like refreshTimeMs?

Reply:
[NIT]

Reply:
I just added  a wait time between retries. Otherwise it was doing retries with very little wait between them.